### PR TITLE
use only 400 and 700 font-weight

### LINKED ISSUE #8060: Wrong font weight across the app?
image
image
- Is that either medium or semibold? 
- IIRC, we used bold on these headings - same as for example, the price
- could you solve it for all the h2 and main headings with this issue?

### PR DESCRIPTION
**Pull request by shashkovdanil**
## PR Type

Bugfix

## Context

Closes #8060 

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP&usdamount=0)

cc @exezbcz

### CODE REVIEW

File: assets/styles/global.scss [hunk 1]
```diff
@@ -50,6 +50,10 @@ body {
     }
   }
 
+  .title {
```
Comment (roiLeo) [inline assets/styles/global.scss:53]: oh no :/ please use text weight helpers instead

File: assets/styles/layouts/_main-navigation.scss [hunk 2]
```diff
@@ -257,7 +257,7 @@
       }
 
       &.is-active {
-        font-weight: bold;
+        font-weight: 700;
```
Comment (roiLeo) [inline assets/styles/layouts/_main-navigation.scss:260]: why change? bold = 700

**Review by roiLeo — COMMENTED** [review]
I think issue comes from #8030 when you changed font import

please use [text-weight](https://bulma.io/documentation/helpers/typography-helpers/#text-weight) helpers when needed

**Review by roiLeo — APPROVED** [review]

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 5.71 USD/DOT ~ 3.503 $DOT
🧗 [16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP](https://polkadot.subscan.io/account/16SpvdDgKNiQ3c53DxX7refnQcKUD3uRNim3Z1HBJLNGrtSP)
🔗 [0x8cee05233ddea4a8a884c85df89cc07df5f3759fd0926bc2fc8f931ae67298b8](https://polkadot.subscan.io/extrinsic/0x8cee05233ddea4a8a884c85df89cc07df5f3759fd0926bc2fc8f931ae67298b8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**